src/content/en/updates/2020/05/devtools.md

```diff
@@ -0,0 +1,126 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: TODO
+
+{# wf_updated_on: 2020-05-11 #}
+{# wf_published_on: 2020-05-11 #}
+{# wf_tags: chrome84, devtools, devtools-whatsnew #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: TODO #}
+{# wf_blink_components: Platform>DevTools #}
+
+# What's New In DevTools (Chrome 84) {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+## Issues panel {: #issues }
+
+The new **Issues** panel was built to help reduce the notifcation fatigue and
```

Comment:
Rename 'Issues panel' to 'Issues tab' (assuming name change goes ahead).

Reply:
FYI, the CL with the rename landed today.

Reply:
👍👏